dsl/src/main/scala/com.crobox.clickhouse/dsl/language/ClickhouseTokenizerModule.scala

```diff
@@ -97,9 +102,14 @@ trait ClickhouseTokenizerModule
     }
   }
 
-  private def tokenizeFinal(asFinal: Boolean): String = if (asFinal) "FINAL" else ""
+  private def tokenizeAs(as: Option[String]): String =
+    as.map {
+        case "FINAL" => "FINAL"
+        case other   => s"AS $other"
```

Comment:
I think it's important to escape this string, so it's SQL safe

Reply:
Fixed escaping (but removed the tokenizeAs as a whole)